### PR TITLE
Autoprefixed css files, adjusted home, sign/login page

### PR DESCRIPTION
**Pull request by MarkellRichards**
Use the auto prefix command within the CSS directory. Short term fix, for now, can automate it later on with scss. 

All the CSS files show they have been changed. That's due to the auto prefix command. Adjusted home, login, sign up, & wish cards pages for cross-platform compatibility

### CODE REVIEW

File: client/public/css/home.css [hunk 1]
```diff
@@ -130,6 +130,7 @@ New color scheme reference is in utils.css
 #home-bg {
   background: url('../img/new-hero.png') center center;
   min-height: 100vh;
+  height: 100px;
```
Comment (stacysealky) [inline client/public/css/home.css:133]: why 100px?

File: client/public/css/stripe.css [hunk 2]
```diff
@@ -60,8 +60,6 @@ input {
   width: 20px;
   height: 20px;
   box-shadow: inset 0 0 0 2px;
-  -webkit-transform: translateZ(0);
```
Comment (stacysealky) [inline client/public/css/stripe.css:63]: why were these deleted?

File: client/public/css/home.css (hunk 1 above)
Comment (MarkellRichards) [reply client/public/css/home.css:133]: That is a hack i found to make IE11 work when justifying content and aligning items center. It will still use the min height bUT needs a height property.

File: client/public/css/stripe.css (hunk 2 above)
Comment (MarkellRichards) [reply client/public/css/stripe.css:63]: Im not sure, maybe they are no longer needed. This wasn't done manually

File: client/public/css/home.css (hunk 1 above)
Comment (stacysealky) [reply client/public/css/home.css:133]: I had no idea! that's cool

**Review by stacysealky — APPROVED** [review]
